wallet/build.gradle

```diff
@@ -105,6 +106,9 @@ ext {
     archivesBaseName = 'dash-wallet'
 }
 
+apply plugin: 'kotlin-android'
```

Comment:
this had to be moved above `android` section in order to be able to specify `jvmTarget` for kotlin (required by some WorkManager related stuff)